multimodal/environment.yml

```
@@ -0,0 +1,10 @@
+name: multimodal
+channels:
+  - defaults
+  - pytorch
+dependencies:
+  - pip=20.1.1
+  - python=3.7.3
+  - pip:
+    # Test requirements
+    - pytest==6.2.2
```

This is out of sync with the test requirements, don't we need mypy and Flake8 as well? One thought is that we could directly use the requirements.txt file here, I believe something like this should work:

```suggestion
  - pip:
    # Test requirements
    - -r requirements_test.txt
```

---

updated and left comments in the file for reference.